cfy_manager/status_reporter/scripts/fill_encoded_user_ids.py

```diff
@@ -0,0 +1,55 @@
+import os
+import json
+import argparse
+
+try:
+    from manager_rest import config
+    from manager_rest.storage import idencoder
+except ImportError:
```

Comment:
After the check you added, this could be removed, no?

Reply:
Yep, you're right